src/server/scripts/EasternKingdoms/Karazhan/karazhan.h

```diff
@@ -23,40 +23,51 @@
 #define KZScriptName "instance_karazhan"
 #define DataHeader "KZ"
 
+#define M_PI_M2       6.28318530718   // 2pi
```

Comment:
Are you sure there isn't an existing M_PI_M2 already ?

Reply:
I'm not sure if something like this doesn't exist, but I couldn't find it